samcli/commands/package/command.py

```diff
@@ -112,8 +131,20 @@ def do_cli(
     metadata,
     region,
     profile,
+    resolve_s3,
 ):
     from samcli.commands.package.package_context import PackageContext
+    from samcli.lib.bootstrap.bootstrap import manage_stack
+    from samcli.commands.deploy import exceptions as deploy_exceptions
```

Comment:
Let's not pull in all deploy exceptions, can we just pull down the single exception? Or move it into the shared exceptions file at `samcli.command.exceptions` since it's shared across methods.